src/sql/src/kafka_util.rs

```diff
@@ -52,22 +52,17 @@ generate_extracted_config!(
     (StartOffset, Vec<i64>)
 );
 
-/// An enum that represents start offsets for a kafka consumer.
-#[derive(Debug)]
-pub enum KafkaStartOffsetType {
-    /// Fully specified, either by the user or generated.
-    StartOffset(Vec<i64>),
-    /// Specified by the user.
-    StartTimestamp(i64),
+impl KafkaConfigOptionExtracted {
+    pub fn group_id_prefix(&self) -> Option<String> {
```

Comment:
The existence of this method seems funny. Seems like the one call site could just do `extracted_config.group_id_prefix`? If that works, saves a clone too; not that it really matters.

Reply:
I have another PR that changes the visibility of the fields (right now it's private) that also makes the change you're suggesting. Agreed on this being goofy.